core/core-dotspacemacs.el

```diff
@@ -140,6 +140,10 @@ with `:' and Emacs commands are executed with `<leader> :'.")
 (defvaralias 'dotspacemacs-remap-Y-to-y$ 'evil-want-Y-yank-to-eol
   "If non nil `Y' is remapped to `y$'.")
 
+(defvaralias 'dotspacemacs-ex-substitute-global 'evil-ex-substitute-global
+  "If non nil, :substitute changes all matches in a line by default.
```

Comment:
It would be better to quote `:substitute` (see other comments for quoting sytle :-) )